.travis.yml

```diff
@@ -6,7 +6,6 @@ addons:
   ssh_known_hosts:
     - github.com 
 node_js:
-  - 6
```

Comment:
We should also update the engine in `package.json` and regenerate package lock file.